Description: Disable bundler as it is provided by the system package.
Author: Lorenzo "Palinuro" Faletra <palinuro@parrotsec.org>
Last-Update: 2017-07-26

--- wpscan-2.9.3.orig/Gemfile
+++ wpscan-2.9.3/Gemfile
@@ -7,9 +7,10 @@ gem 'yajl-ruby', '>=1.3.0' # Better JSON
 gem 'terminal-table', '>=1.6.0'
 gem 'ruby-progressbar', '>=1.8.1'
 
-group :test do
-  gem 'webmock', '>=2.3.2'
-  gem 'simplecov', '>=0.13.0'
-  gem 'rspec', '>=3.5.0'
-  gem 'rspec-its', '>=1.2.0'
-end
+# Disabled so that we can run bundle install without those libs
+#group :test do
+#  gem 'webmock', '>=2.3.2'
+#  gem 'simplecov', '>=0.13.0'
+#  gem 'rspec', '>=3.5.0'
+#  gem 'rspec-its', '>=1.2.0'
+#end
--- wpscan-2.9.3.orig/lib/environment.rb
+++ wpscan-2.9.3/lib/environment.rb
@@ -15,7 +15,7 @@ Encoding.default_external = Encoding::UT
 begin
   # Standard libs
   require 'readline'
-  require 'bundler/setup' unless kali_linux?
+#  require 'bundler/setup' unless kali_linux? Disabled for Debian, we have proper dependencies
   require 'getoptlong'
   require 'optparse' # Will replace getoptlong
   require 'uri'
